build_releases

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -eu
+
+NAME="ignition"
+ORG_PATH="github.com/coreos"
+REPO_PATH="${ORG_PATH}/${NAME}"
+VERSION=$(git describe --dirty)
+GLDFLAGS=${GLDFLAGS:-}
+GLDFLAGS+="-X github.com/coreos/ignition/internal/version.Raw=${VERSION}"
+
+if [[ -n "$(git status -s)" ]]; then
```

Comment:
This isn't sufficient. It will still succeed when there are untracked files.

Reply:
Should it just run `git clean -f` rather than checking if the files exist?

Reply:
You probably don't want it destroying anything that might be of use.

Reply:
I don't think that untracked files should have any bearing on the ability to run a release build, as long as all the files in the repository are committed.

Reply:
also it won't succeed if there are untracked files?

```
│siril|master|~/work/coreos/ignition
│> git status
On branch master
Your branch is up to date with 'upstream/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	a

nothing added to commit but untracked files present (use "git add" to track)
│siril|master|~/work/coreos/ignition
│> git status -s
?? a

```

Reply:
Confirming with the script as well, the behavior of `git status -s` is such that it reports untracked files, and the script will thus refuse to run if any exist.

```
derek@proton ~/go/src/github.com/coreos/ignition> ./build_releases 
building ignition-validate-x86_64-unknown-linux-gnu
building ignition-validate-x86_64-apple-darwin
building ignition-validate-x86_64-pc-windows-gnu.exe
building ignition-validate-aarch64-unknown-linux-gnu
./build_releases  59.41s user 3.71s system 237% cpu 26.547 total
derek@proton ~/go/src/github.com/coreos/ignition> touch untracked-file
derek@proton ~/go/src/github.com/coreos/ignition> ./build_releases    
git repo not clean
```

Reply:
Ohhhhhh, I read that too quickly. I thought it was only looking at the exit code.